styles/components/_pageHeader.scss

```diff
@@ -34,6 +34,16 @@ $z-header: 100;
     @include for-tablet-landscape-up {
       flex-wrap: nowrap;
     }
+    hr {
+      display: blocks;
+      background-color: rgba(223, 225, 226, 0.25);
```

Comment:
is there a reason that you're using the rgb function here versus a color variable? i'm not saying you should be using a variable here i'm just curious